WordPressAuthenticator/WordPressAuthenticator/NUX/WPHelpIndicatorView.swift

```diff
@@ -0,0 +1,36 @@
+import UIKit
+import WordPressShared
+
+open class WPHelpIndicatorView: UIView {
+
+    struct Constants {
+        static let defaultInsets = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 0)
```

Comment:
Tip tip! this is equivalent to `UIEdgeInsets.zero`.

There's also `CGRect.zero` and `CGPoint.zero`. Neat!!

Reply:
Ah, good catch. It shall be changed!